examples/accept.yaml

```diff
@@ -0,0 +1,95 @@
+programs:
+  - name: accept
+    metrics:
+      histograms:
+        - name: accept_latency_seconds
+          help: Latency histogram for how long sockets sit in accept queue
+          table: accept_latency
+          bucket_type: exp2
+          bucket_min: 0
+          bucket_max: 26
+          bucket_multiplier: 0.000001 # microseconds to seconds
+          labels:
+            - name: port
+              size: 8
+              decoders:
+                - name: uint
+            - name: bucket
+              size: 8
+              decoders:
+                - name: uint
+    kprobes:
+      inet_csk_reqsk_queue_add: trace_inet_csk_reqsk_queue_add_start
+      inet_csk_accept: trace_inet_csk_accept_start
+    code: |
+      // Fix build error because of missing "KBUILD_MODNAME"
+      #define KBUILD_MODNAME "foo"
+
+      #include <uapi/linux/ptrace.h>
+      #include <net/sock.h>
+      #include <net/request_sock.h>
+      #include <net/inet_connection_sock.h>
+      #include <bcc/proto.h>
+
+      typedef struct listen_socket_key {
+          u16 port;
+          u64 slot;
+      } listen_socket_t;
+
+      // 27 buckets for latency, max range is 33.6s .. 67.1s
+      const u8 max_latency_slot = 26;
+
+      // max number of listening ports we expect to see on the host
+      const u32 max_ports = 1024;
+
+      // Histograms to record latencies
+      BPF_HISTOGRAM(accept_latency, listen_socket_t, (max_latency_slot + 1) * max_ports);
+
+      // Sockets to start time map
+      BPF_HASH(start, struct request_sock *, u64);
+
+      int trace_inet_csk_reqsk_queue_add_start(struct pt_regs *ctx, struct sock *sk, struct request_sock *req)
+      {
+          u64 ts = bpf_ktime_get_ns();
+          start.update(&req, &ts);
+          return 0;
+      }
+
+      int trace_inet_csk_accept_start(struct pt_regs *ctx, struct sock *sk)
+      {
```

Comment:
Other examples keep `{` at the same line with the function signature, let's keep it that way for consistency.